schemas/schemas.go

```diff
@@ -0,0 +1,36 @@
+package schemas
+
+const ReferencePlan = `{
```

Comment:
Where are the other properties of the plan being defined?

`referencePlan.ID = UUID.String()`
`referencePlan.CatalogID = UUID.String()`
`referencePlan.CatalogName = instance_sharing.ReferencePlanName`
`referencePlan.Name = instance_sharing.ReferencePlanName`
`referencePlan.Description = instance_sharing.ReferencePlanDescription`
`referencePlan.ServiceOfferingID = serviceOfferingId`
`referencePlan.Bindable = newTrue()`
`referencePlan.Ready = true`
`referencePlan.CreatedAt = time.Now()`
`referencePlan.UpdatedAt = time.Now()`

Reply:
Saw now, on `generateReferencePlanObject`

why do we split them? 
won't it be better to have everything in the same place?